.travis.yml

```diff
@@ -12,7 +12,7 @@ install: true
 script: "./travis.sh"
 env:
 - TEST=CI
-- TEST=plugin SQ_VERSION=LTS
+- TEST=plugin SQ_VERSION=5.6-RC2
```

Comment:
Are we supposed to merge the PR with that? We will have to change it as soon as LTS is released.